purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -594,6 +599,10 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
      * @param newAppUserID The new appUserID that should be linked to the currently user
      * @param [listener] An optional listener to listen for successes or errors.
      */
+    @Deprecated(
+            "Use logIn instead",
+            ReplaceWith("logIn")
```

Comment:
You can add the parameters of the `logIn` function that way the developers get a nice automatic "replace with" option in Android Studio. Maybe it works just specifying the function name, but I don't think so...

Reply:
nice, didn't know that was a thing, will do!